packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -2,13 +2,13 @@
 title: Overview of the mobile Identity Engine SDK
 ---
 
-Design the implementation of the sign-in flow for your mobile app by understanding the representation of the flow and the associated objects.
+Design the sign-in implementation for your mobile app by understanding the objects and messages that represent the sign-in flow.
 
 <ApiLifecycle access="ie" /><br>
 
 ## Introduction
 
-Integrate the Okta sign-in flow into your app in one of two ways:
+There are two ways to use Okta services for user sign-in in your app:
```

Comment:
sign-in isn't a noun
```suggestion
There are two ways to use Okta services to sign a user in to your app:
```

Reply:
Made the change. It's in the next commit.